### PR TITLE
delete files that are listed in .gitignore (html and auxillary), update README

### PR DESCRIPTION
**Pull request by sumny**
For consistency with the current workflow, delete all files that are listed in .gitignore (importantly html and auxiliary files that were generated when rendering a local preview). Also updated REAMDE to make contributors aware of this.

### CODE REVIEW

**Comment by jakob-r** [comment]
There are still some html files that gets changed with this PR. This seems to be wrong?

**Comment by sumny** [comment]
> There are still some html files that gets changed with this PR. This seems to be wrong?

Yes :( seems like this is not the real solution. I tried to automate it calling `git ls-files -i --exclude-from=.gitignore` and then removing these via `git rm --cache`.

**Comment by sumny** [comment]
@jakob-r I guess it looks better now?

**Comment by sumny** [comment]
So the problem is that the CI automatically re-renders the posts (Deploy from Github Actions build 104717928 [ci skip]) and therefore the html files in `docs/posts` are changed?

**Comment by jakob-r** [comment]
Yes. Is that intended?

**Comment by sumny** [comment]
> Yes. Is that intended?

I guess so, at least in the README it states: "All posts are rendered on every push to ensure that all posts can be run without issues."

**Comment by sumny** [comment]
> > Yes. Is that intended?
> 
> I guess so, at least in the README it states: "All posts are rendered on every push to ensure that all posts can be run without issues."

@pat-s do you have any comments here?

File: README.md [hunk 1]
```diff
@@ -19,12 +19,12 @@
    ```
    ````
 1. Write the post and select appropriate categories. Tags are not available.
-1. Add required packages to DESCRIPTION file via `usethis::use_package(<package>)`.
-1. Preview locally using `rmarkdown::render_site()` and `pkgdown::preview_site()` or (when using RStudio) by clicking on "Build Website" in the "Build" pane.
```
Comment (pat-s) [inline README.md:23]: Why should this be removed? This should still work and the way to do it to preview a draft post if a local Rmd has been rendered.

File: README.md [hunk 2]
```diff
@@ -19,12 +19,12 @@
...
+1. Add required packages to the DESCRIPTION file via `usethis::use_package(<package>)`.
+1. Preview your post locally using `rmarkdown::render()`.
```
Comment (pat-s) [inline README.md]: Hm no, this was not what I meant.

1. Render the Rmd to create the HTML
1. Preview via `rmarkdown::render_site()` and `pkgdown::preview_site()`

**Review by pat-s — CHANGES_REQUESTED** [review]
`docs/` always changes since all posts are rerun on every push and deployed into `docs/`.
To have a preview of the post (which was Bernds wish) this is required.

What you see locally in the preview when running `rmarkdown::render_site()` is what's in `_posts/**/.html`.

If one wants to draft a new post, just push the Rmd, let GHA create all auxillary files and don't worry about them.
They will match with `master` after the PR was merged. 

If master is merged into a PR, GHA will update the auxillary files automatically in the PR.

After all these issues and confusion I regret having made the switch to distill :/

File: README.md (hunk 1 above)
Comment (sumny) [reply README.md:23]: yes, makes sense. Fixed this.

File: README.md (hunk 2 above)
Comment (sumny) [reply README.md]: I guess I missunderstood you here. I reverted these parts of the README.

**Comment by pat-s** [comment]
Ok, let's see what happens to master after merge.

**Pull request merged**